spec/02-integration/03-admin_api/09-targets_routes_spec.lua

```diff
@@ -257,6 +257,65 @@ describe("Admin API", function()
           assert.equal([[{"data":[],"total":0}]], body)
         end)
       end)
+
+      describe("active targets", function()
+        local upstream_name3 = "example.com"
+
+        before_each(function()
+          local upstream3 = assert(helpers.dao.upstreams:insert {
+            name = upstream_name3,
+          })
+
+          -- two target inserts, resulting in a 'down' target
+          assert(helpers.dao.targets:insert {
+            target = "api-1:80",
+            weight = 10,
+            upstream_id = upstream3.id,
+          })
+          assert(helpers.dao.targets:insert {
+            target = "api-1:80",
+            weight = 0,
+            upstream_id = upstream3.id,
+          })
+
+          -- three target inserts, resulting in an 'up' target
+          assert(helpers.dao.targets:insert {
+            target = "api-2:80",
+            weight = 10,
+            upstream_id = upstream3.id,
+          })
+          assert(helpers.dao.targets:insert {
+            target = "api-2:80",
+            weight = 0,
+            upstream_id = upstream3.id,
+          })
+          assert(helpers.dao.targets:insert {
+            target = "api-2:80",
+            weight = 10,
+            upstream_id = upstream3.id,
+          })
+
+          -- and an insert of a separate active target
+          assert(helpers.dao.targets:insert {
+            target = "api-3:80",
+            weight = 10,
+            upstream_id = upstream3.id,
+          })
+        end)
+
+        it("only shows active targets", function()
+          local res = assert(client:send {
+            method = "GET",
+            path = "/upstreams/"..upstream_name3.."/targets/active/",
```

Comment:
style: ideally - even if maybe this file is not up to date - this should have spaces around the `..` operator